argocd/efs-provisioner/values.yaml

```diff
@@ -1,2 +1,3 @@
----
-efsProvisioner: {}
+# ---
+# # -- Values passed to the efs-provisioner chart
+# efsProvisioner: {}
```

Comment:
Are you sure we should comment the default?

Reply:
reverted to master